src/controllers/health/index.js

```diff
@@ -11,26 +10,6 @@ routes.get('/cra0h', (req, res, next) => {
   return next(new Error('crash test'));
 });
 
-routes.get('/trace', (req, res, next) => {
```

Comment:
As far as I know this endpoint is unused, but I'm not sure what it was ever used for.